.github/workflows/preview-build.yml

```
@@ -0,0 +1,16 @@
+    branches:
+      - preview
+
+jobs:
+  build:
+    name: Build and publish preview container image
+    uses: ./.github/workflows/_build.yml
+    with:
+      docker_tags: |
+        type=raw,value=preview
+    secrets:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.12)

[error] secret "GITHUB_TOKEN" is not defined in "./.github/workflows/_build.yml" reusable workflow. no secret is defined

(workflow-call)